web-app/src/i18n/en-us/tournament.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  name: 'Tournament Name',
+  team: 'Team',
+  teams: 'Teams List',
+  teamName: 'Team name',
+  teamCapitan: 'Captian',
+  teamSecondMember: 'Second player',
+  teamSecondMemberCreator: 'Secobd player (optional)',
+  teamAdd: 'Add team',
```

Comment:
```suggestion
  addTeam: 'Add team',
```
hmm, what do you think?

Reply:
changing to team.add (as above)